fuse-products/src/main/java/software/tnb/product/deploystrategy/OpenshiftBaseDeployer.java

```diff
@@ -121,7 +123,10 @@ protected void copyResources(Path contextPath, String destinationFolder) {
                     resources.forEach(resource -> {
                         try {
                             if (resource.getIsContentPath()) {
-                                Files.copy(Paths.get(resource.getContent()), resFolder.resolve(resource.getName()));
+                                //to create target path if it does not exist:
+                                IOUtils.writeFile(resFolder.resolve(resource.getName()), "fake-content");
```

Comment:
is it necessary?

Reply:
yes - if you want to use file structure (like resFolder/folder1/file), ie not to copy files directly to the resFolder.

Reply:
what do you think something like ?`Files.createDirectories(resFolder.resolve(resource.getName()).getParent());`

Reply:
yes, indeed.

Reply:
@Stefan365 fell free to add a new method (eg: `copyFile`) in `software.tnb.common.utils.IOUtils` that you can use also in `IntegrationGenerator`

Reply:
ok

Reply:
done